es-core/src/Settings.cpp

```diff
@@ -47,6 +48,7 @@ void Settings::setDefaults()
 	mBoolMap["ShowExit"] = true;
 	mBoolMap["Windowed"] = false;
 	mBoolMap["SplashScreen"] = true;
+	mStringMap["System"] = "";
```

Comment:
I think the name ` System` should be more descriptive. Maybe `StartupSystem`, or even `SystemToShowAtStartup` ?